src/App.js

```diff
@@ -1,15 +1,57 @@
-import React from "react";
+import React, {useState, useEffect} from "react";
 import "./App.css";
+import axios from "axios";
+
+export default function App() {
+
+  const [state, setState] = useState({
+    date: "",
+    explanation: "",
+    media_type: "",
```

Comment:
Great use of the use state pattern here rather than doing it all with independent use state calls. This way it seems more similar to what are used to using.